### PR TITLE
Alerting: Fix API specification

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:
Fix specification error:
```
Semantic error at paths./api/alertmanager/{Recipient}/config/api/v1/receivers/test.post.parameters
Multiple body parameters are not allowed.
Jump to line 3731
```

The updated specification:
https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/papagian/ngalert-fix-api-spec/pkg/services/ngalert/api/tooling/spec.json

**Which issue(s) this PR fixes**:

Fixes grafana/technical-documentation#133

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ngalert/api/tooling/definitions/api.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
+// Grafana 8 Alerts API.
+//
 // Package definitions includes the types required for generating or consuming an OpenAPI
-// spec for the Unified Alerting API.
-// Documentation of the API.
+// spec for the Grafana 8 Alerts API.
```
Comment (yuri-tceretian) [inline pkg/services/ngalert/api/tooling/definitions/api.go:4]: We call it Grafana Alerting now.

**Review by yuri-tceretian — APPROVED** [review]
LGTM

**Comment by armandgrillet** [comment]
Thanks for this PR, should we backport it to have it as part of 8.3.1?

**Comment by papagian** [comment]
> Thanks for this PR, should we backport it to have it as part of 8.3.1?

AFAIK we don't publish this API except from [here](https://grafana.com/docs/grafana/latest/http_api/alerting/) and it points to the version in the `main` branch. So I don't think we need a backport.

**Pull request merged**